### ISSUE TITLE
Google Play Music responds incorrectly to the Play/Pause and Stop media keys

### ISSUE THREAD

**Issue by mountainpenguin** [tiliado/nuvolaplayer#171]
Google Play Music plays a random song when pressing the Play/Pause media key instead of playing/pausing. The Stop media key appears to have no effect. The previous and next song media keys function as expected.

System information:
Distro: Arch Linux (up-to-date as of October 27, 2015)
Kernel: 4.2.4-1-ARCH
Architecture: x86_64
Desktop: GNOME Shell 3.18.1

diorite version as of commit tiliado/diorite@5aa92737af32746cc8597c759e7bfc9752fd0c56
nuvolaplayer version as of commit e5855445cfb0e44309db8938fa41f5c7c741a7e3

Both diorite and nuvolaplayer built with the following commands:
```
./waf configure --prefix=/usr
./waf build
sudo ./waf install
```

nuvola-app-google-play commit tiliado/nuvola-app-google-play@8ca0bad8690e662a52986a60d675db33689e326b
Built with the following commands:
```
make build
make install
```

Steps taken to reproduce the issue (and produce the attached log file):
1. Start nuvolaplayer with `nuvolaplayer3 -D`
2. Select Google Play Music
3. Start a playlist
4. Press the Play/Pause media key (selects a random song)
5. Press the Stop media key (does nothing)
6. Press the Next media key (works as expected)
7. Press the Previous media key (works as expected)

Log file is attached at https://gist.github.com/mountainpenguin/8b876d5e5953f63359a0
I have separated each step in the above list with linebreaks.

Sorry if this has already been reported previously, I couldn't see anything from a quick search. And let me know if there is any other information I can provide.

Thanks!

**Comment by DidierLoiseau** [comment]
I observe the same issue on Ubuntu using Nuvola installed through apt.

    $ nuvolaplayer3 --version
    Nuvola Player 3 Beta 3.0.0+201510101914.d95d0c6
    Revision d95d0c6d446c3fd32516d7458930b563eb31da32

Using the play/pause button in GMusic works though – only way to actually pause the music.

**Comment by fenryxo** [comment]
Hi. Thanks for your report. It seems that Google Play Music web interface has changed and integration script has to be updated.

**Comment by fenryxo** [comment]
Fixed in Google Play Music script 5.8.
